_posts/2023-01-01-announcing-djangocon-europe-2023.md

```diff
@@ -0,0 +1,34 @@
+---
+author: DjangoCon Europe Organisers
+category: General
+date: 2023-01-01 06:00:00
+layout: post
+published: true
+post_photo_alt: The view of Edinburgh from Calton Hill
+post_photo_url: /static/img/locations/calton-hill.jpg
+title: DjangoCon Europe is coming to Edinburgh!
+---
+
+Hello, everyone!  This year, DjangoCon Europe is coming to Edinburgh, Scotland!
+
+We are very excited to announce that the website for DjangoCon Europe 2023 is [live](https://2023.djangocon.eu).
+
+Many thanks to [TorchBox](https://torchbox.com/) for the website design!
+
+The conference will be from Monday 29th May to Friday 2nd June 2023.
+
+- 29-31 May: Talks, Tutorials and Workshops
+- 1-2 June: Sprints
+
+Talks, tutorials and workshops will be held at the [Assembly Rooms](/venues/#talks), in 
+the heart of Edinburgh's city centre. Sprints will be held at the 
+[Biscuit Factory](/venues/#sprints) in Leith.
```

Comment:
I see we have an issue with the site stylesheet adapting to those anchor links. Will change the CSS.